src/sidebar/components/MarkdownEditor.tsx

```diff
@@ -494,6 +505,15 @@ function Toolbar({
         shortcutKey={SHORTCUT_KEYS.list}
         title="Bulleted list"
       />
+      {mentionsEnabled && (
+        <ToolbarButton
+          disabled={isPreviewing}
+          icon={MentionIcon}
+          onClick={() => onCommand('mention')}
+          title="Mention"
+          classes="!text-brand"
```

Comment:
The use of the brand color is just to highlight that this is a new feature, right? Can you note that in a comment.